star_tides/exceptions.py

```diff
@@ -0,0 +1,104 @@
+''' star_tides.exceptions.base_exception
+'''
+
+# pragma pylint: disable=E1124
+
+from http import HTTPStatus
+
+
+# LOG LEVELS
+SEVERITY_DEBUG = 'DEBUG'
```

Comment:
Up to you: make this `class Severity(Enum): ...`?

Reply:
I'd prefer not to because it will be a part of the logging message.